docs/configuration/http_conn_man/route_config/vhost.rst

```diff
@@ -53,3 +54,7 @@ require_ssl
 :ref:`virtual_clusters <config_http_conn_man_route_table_vcluster>`
   *(optional, array)* A list of virtual clusters defined for this virtual host. Virtual clusters
   are used for additional statistics gathering.
+
+:ref:`rate_limits <config_http_conn_man_route_table_rate_limit_config>`
+  *(optional, array)* Specifies a set of rate limit configurations that could be applied to the
```

Comment:
"that will be applied" also please link to rate limit arch overview / other applicable docs